docs/examples/example.ipynb

```diff
@@ -417,48 +389,164 @@
     "exec python3 <your-module-directory-name>.<main-program-filename-without-extension> $@\n",
     "```\n",
     "\n",
-    "In this case, `<your-modular-directory-name>.<main-program-filename-without-extension>` would be replaced with `src.main`. \n",
-    "\n",
     "To make this shell script executable, run this in the Terminal:\n",
     "\n",
-    "```sudo chmod +x <FILEPATH>/exec.sh```\n",
+    "```sudo chmod +x <FILEPATH>/<FILENAME>```\n",
+    "\n",
+    "In this case, `<your-module-directory-name>.<main-program-filename-without-extension>` would be replaced with `src.main`. `<FILENAME>` would be `exec.sh`.\n",
```

Comment:
The first sentence of this callout belongs beneath the `exec` above the `chmod` instead.

Is the `<FILENAME>` always going to be `exec.sh` ? If, I recommend removing the second sentence advising such, and updating the example directly:

```
"```sudo chmod +x <FILEPATH>/exec.sh```\n",
```

Reply:
It will not always be `exec.sh`. For example, the module example calls it `run.sh`. The naming is up to the user.